### ISSUE TITLE
search artists page

### ISSUE THREAD

**Issue by jeffbum** [Artist-Assemble/Artist-Assemble#76]
How do I look at someone else's profile and see all of their tracks? Or is this not a feature?

**Comment by jeffbum** [comment]
oops. my mistake. I found it. maybe a hover feature that pops up text explaining what the arrow button means.

**Comment by gabe1331** [comment]
yes, easy addition. I'll add that. Otherwise, I ultimately want to remove the botton and just have an arrow while they can click on the whole user container, but it overrides the pause and play button. If you have a solution let me know
